selftests/vmimage/tests/plugin/unit_plugin_vmimage.py

```diff
@@ -1,14 +1,32 @@
+import json
 import os
 import tempfile
 import unittest.mock
 from urllib.error import URLError
 
 from avocado.core.settings import Settings
 from avocado.plugins import vmimage as vmimage_plugin
+from avocado.utils import path
 from avocado.utils import vmimage as vmimage_util
-from selftests.functional.plugin.vmimage import create_metadata_file, missing_binary
 from selftests.utils import skipOnLevelsInferiorThan, temp_dir_prefix
 
+
+def missing_binary(binary):
```

Comment:
Maybe I am missing something here, but what is the reason for copying these methods instead of importing them?